dfastmi/Dutch_rivers_v3.ini

```diff
@@ -1,8 +1,8 @@
 [General]
-    Version    = 2.0
+    Version    = 3.0
```

Comment:
What is the difference in version that it needs to be updated to v3?

Reply:
@ralph-peelen-deltares 
Version 3 is identical to version 2, except for the version number.
It was decided to change this to 3 to keep the file version in line with the application version.

Reply:
Discussed with Ralph.